src/components/dropdownSearch/MultiSelectDropdownSearch/MultiSelectDropdownSearch.tsx

```diff
@@ -10,8 +10,6 @@ export class MultiSelectDropdownSearch extends DropdownSearch {
     createOptionText: 'Create option for ',
     isOpened: false,
     options: [],
-    displayedOptions: [],
-    selectedOptions: [],
```

Comment:
Is this part of another feature/fix or these are simply not used?

Reply:
Interface problem with typescript, fix incoming